engines/router/missionctl/server/application.go

```diff
@@ -67,26 +67,31 @@ func Run() {
 		}
 
 		s := upi.NewUPIServer(missionCtl)
-		if cfg.AppConfig.CustomMetrics {
-			m := cmux.New(l)
-			grpcL := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
-			httpL := m.Match(cmux.HTTP1Fast())
+		m := cmux.New(l)
+		grpcL := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
```

Comment:
@leonlnj Is the `content-type` case-sensitive? I was wondering whether the request to metrics server routed to incorrect handler due to this.

Reply:
yes it is but from what i searched and learnt, http2 headers are all lower-case before encoded by design, so it should be safe.

https://www.rfc-editor.org/rfc/rfc7540#section-8.1.2
```
 Just as in HTTP/1.x, header field names are strings of ASCII
 characters that are compared in a case-insensitive fashion.  However,
 header field names MUST be converted to lowercase prior to their
 encoding in HTTP/2.
```
https://evertpot.com/http-2-finalized/